### ISSUE TITLE
BotSentry incompatibility with LimboAPI 

### ISSUE THREAD

**Issue by pVICTORIAAAA** [BlackDemonZyT/BotSentry#125]
Velocity plugin incompatibility with LimboAPI & BotSentry (Yeah for bug you need plugin that uses LimboAPI for example LimboAUTH cuz its verification plugin)

So yeah basically plugins works perfectly until you take a look at botsentry what it does;
- Doesn't blacklist any players with basic configuartion which is given by botsentry plugin (Yes it shows player uses proxy/vpn on /bs scan command)
- Doesn't have first join kick which should be there

So yeah, starting with no errors console shows perfectly that everything works even under /velocity plugins
velocity ver is 3.3.0 - snapshot | LimboAPI is 1.1.24 

Reproduction of the issue is kinda easy; you just need to start Velocity server with plugins LimboAPI + LimboAUTH for example and BotSentry, it shows that it run perfectly but it for sure don't tested with public proxies which it detected, even I blacklisted myself but yeah, still I could join without hesitation. Hope it get Fixed ASAP. 

![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/11975931-f723-4b18-a059-a8346971ba12)

! CONFIG FILE FOR BOTSENTRY IS CLEARED EVERY SCREENSHOT !
That's how it should look like and it is great until u use one of plugin that uses library which is limboapi + limboauth for example
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/aa391e4a-da0a-4834-9f46-018c249e06eb)

And with limboauth, there's no display for that kick message anymore
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/90455a50-37ee-4c43-88c8-bb3c5310c151)

**Comment by Laurenshup** [comment]
Okay, to be sure. Players are not being kicked when you have LimboAPI and LimboAuth active. Could you maybe send me a screenshot of /bs scan of a player using a VPN/proxy? I want to check if it actually adds it to the blacklist or not.

**Comment by pVICTORIAAAA** [comment]
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/95e16d1e-1a15-4394-9e4c-053cefa2e68b)
Here you go 
+ yes no blacklist
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/408d12e6-e93d-44ad-9faf-742de02fd81e)

Okay Thats fucking weird, I opened the blacklist.json

![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/85abdb44-6f04-4d00-8553-f2693e62de6b)

Tried add by command but still can even join
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/6cbf8542-3217-4d9b-956c-abba44c6649e)

okay nvm it added after restart but it doesn't flag me :v
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/90d452bb-4096-4211-9697-43372f6d6fa6)

yeah no kick F
![image](https://github.com/BlackDemonZyT/BotSentry/assets/97870251/93fff51b-0a76-4196-943f-edfce6f97ad9)

**Comment by Laurenshup** [comment]
Okay thanks, looks like internally BotSentry does correctly flag it as disconnected. I am not sure what LimboAPI and LimboAuth are actually about, as it looks like it changes the functions of Velocity that disconnects the player. Do both plugins cooperate together, are both needed to let this happen?

**Comment by pVICTORIAAAA** [comment]
Yes, limboapi is created and maintained by same developer and they are open source on GitHub also they are needed to be together to botsentry stop working correctly.

As provided information before ur latest response u got screenshot with botsentry kick which was tested with only limboapi and botsentry but it works fine, hope it helps you wish u best, goodnight!

**Comment by pVICTORIAAAA** [comment]
This link is to author of limboapi and limboauth you could view source of their plugin here:
https://github.com/Elytrium/LimboAuth
https://github.com/Elytrium/LimboAPI
https://github.com/Elytrium

**Comment by Laurenshup** [comment]
Okay, so I am going to simulate LimboAuth and LimboAPI on my testing server, do you have any special settings I should test with?

**Comment by pVICTORIAAAA** [comment]
> Okay, so I am going to simulate LimboAuth and LimboAPI on my testing server, do you have any special settings I should test with?

No, just run it default settings you can change anything u want doesn't work any option for me
